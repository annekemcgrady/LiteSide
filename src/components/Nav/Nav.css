.nav {
  border: 1px solid red;
  height: 90vh;
  width: 16%;
}