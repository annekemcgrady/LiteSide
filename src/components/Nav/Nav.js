import React from 'react';
import { NavLink } from 'react-router-dom';
import './Nav.css';
import logo from '../../images/logo.png';

const Nav = ({favCount}) => {

  return (
    <nav className="nav">
        <img className="logo" src={logo} alt="lite side logo"/>
        <div className="buttons-container">
          <NavLink to='/people' className="nav-btn" activeClassName="active-btn" >PEOPLE</NavLink>
          <NavLink to='/planets' className="nav-btn" activeClassName="active-btn">PLANETS</NavLink>
          <NavLink to='/vehicles' className="nav-btn" activeClassName="active-btn">VEHICLES</NavLink>
          <div className="fav-btn-container">
            <p className="fav-count">{favCount}</p>
            <NavLink to="/favorites" className="nav-fav-btn" activeClassName="fav-active">FAVORITES</NavLink>
          </div>
        </div>
    </nav>
  )
}

export default Nav;
