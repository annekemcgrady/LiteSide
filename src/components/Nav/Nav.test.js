import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav';

describe('Nav', ()=> {
  it('should match the snapshot with all data passed in correctly', () => {

    const wrapper = shallow( <Nav  />)

    expect(wrapper).toMatchSnapshot();
  });



});

