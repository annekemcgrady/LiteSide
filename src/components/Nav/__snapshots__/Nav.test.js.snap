// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot with all data passed in correctly 1`] = `
<nav
  className="nav"
>
  <img
    alt="lite side logo"
    className="logo"
    src="logo.png"
  />
  <div
    className="buttons-container"
  >
    <NavLink
      activeClassName="active-btn"
      className="nav-btn"
      to="/people"
    >
      PEOPLE
    </NavLink>
    <NavLink
      activeClassName="active-btn"
      className="nav-btn"
      to="/planets"
    >
      PLANETS
    </NavLink>
    <NavLink
      activeClassName="active-btn"
      className="nav-btn"
      to="/vehicles"
    >
      VEHICLES
    </NavLink>
    <div
      className="fav-btn-container"
    >
      <p
        className="fav-count"
      />
      <NavLink
        activeClassName="fav-active"
        className="nav-fav-btn"
        to="/favorites"
      >
        FAVORITES
      </NavLink>
    </div>
  </div>
</nav>
`;
